### DISCUSSION TITLE
New map shapes

### DISCUSSION THREAD

**Discussion by TheLarry98** [OpenRCT2/OpenRCT2#15768] [Feature requests]
Currently, you only can set a square shaped map. 
 (e.g. 160 x 160)
It would be great when we could make like a rectangle shaped map. (e.g. 80 x 120)

But much better would be when we could shape the map with the landtool itself so that some parts of the map look bigger and some parts smaller.  Guys on neddesign already do this but in another way.
Take a look at this:
![openrct2 different shaped map](https://user-images.githubusercontent.com/12634026/28336418-2c1ad26c-6c02-11e7-9f3d-7f099b5a40cb.png)
The park shown on the screenshot is Six Flags Santa Fe which can downloaded here:
https://www.nedesigns.com/park/1925/six-flags-santa-fe/

I hope this would be possible! Thanks.

**Comment by spacek531** [comment]
You cannot hide tiles with the tile inspector. What happens is a void where no rendering takes place - whatever was rendered above the void is what is displayed.
![image](https://user-images.githubusercontent.com/3454156/28348930-9fe10ff8-6bf4-11e7-8dba-cb587a8428ec.png)
